assets/scripts_ssh_config

```diff
@@ -1,3 +1,4 @@
 StrictHostKeyChecking no
 UserKnownHostsFile /dev/null
+PasswordAuthentication no
```

Comment:
mmm, I think that means the config drive isn't working right - the ssh key should get installed, and that's how the WaitForSSH gets in.

kinda very weird that it works for the iso version of this test.